guardian/recipes-backend
# Insert article time data into recipe model

## What does this change?

Extracts time data from the article and inserts into the recipe model, so that it can be used in the recipe search.  

It's currently unclear which time data will be most useful and so, for now, the following fields are being extracted:
- lastModifiedDate
- firstPublishedDate
- publishedDate

## How to test

Run `npm test` - tests pass

Deploy to CODE
Find a recipe article on the Guardian website and use `Teleporter | flexible composer | Management | View previous versions` to `Restore` the recipe to CODE
Access `recipe-responder` in ElasticSearch and filter to CODE - this will check it hasn't broken
Locate the DEBUG message with `urlToPurge` that does **not** relate to index.json, and extract the part of the url from `recipes.code` onwards
Append this to `https://` in a new browser window
View the output and check that the following have been populated:
- lastModifiedDate
- firstPublishedDate
- publishedDate

## How can we measure success?



## Have we considered potential risks?



## Images



## Accessibility
